pages/settings/settings.js

```diff
@@ -135,6 +136,12 @@ trackerCheckbox.addEventListener("change", function (e) {
 	});
 });
 
+// Dark Mode Settings
+darkModeCheckbox.addEventListener("change", function (e) {
+	settings.set("darkMode", this.value)
```

Comment:
The ```value``` of a checkbox is always "on", whether the checkbox is checked or not, so this should be ```this.checked``` instead.